cmd/lint/runner.go

```diff
@@ -0,0 +1,111 @@
+package lint
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"sort"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/cobra"
+
+	"github.com/giantswarm/config-controller/pkg/generator"
+	"github.com/giantswarm/config-controller/pkg/github"
+	"github.com/giantswarm/config-controller/pkg/lint"
+)
+
+const (
+	owner = "giantswarm"
+	repo  = "config"
+)
+
+type runner struct {
+	flag   *flag
+	logger micrologger.Logger
+	stdout io.Writer
+	stderr io.Writer
+}
+
+func (r *runner) Run(cmd *cobra.Command, args []string) error {
+	ctx := context.Background()
+
+	err := r.flag.Validate()
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	err = r.run(ctx, cmd, args)
+	if IsLinterFoundIssues(err) {
```

Comment:
We should not change `Run`. Long-term that will be generated.

Why this error should not be printed?